### ISSUE TITLE
Migrate testing from `sass` to `sassc`

### ISSUE THREAD

**Issue by jodosha** [hanami/hanami#955]
```shell
Post-install message from sass:

Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  http://sass.logdown.com/posts/7081811
```

**Comment by aitbw** [comment]
Hi! I'd love to tackle this issue, is it still available?

**Comment by AlfonsoUceda** [comment]
@aitbw please go ahead, thank you so much!

**Comment by lfmalves** [comment]
@aitbw, any news on this front?

**Comment by landongrindheim** [comment]
@jodosha This is closed by #1011. #979 can also be closed 😄

**Comment by landongrindheim** [comment]
@jodosha This is closed by #1011. #979 and #1010 can also be closed 😄
